package.json

```diff
@@ -56,8 +58,7 @@
   },
   "dependencies": {
     "@placemarkio/polyline": "^1.2.0",
-    "nanoid": "^4.0.2",
-    "svelte-spa-router": "^3.3.0"
```

Comment:
Why moving the router to dev dependencies?

Reply:
Because it's used only in demo app and isn't needed for the shipped lib

Reply:
But the demo app gets deployed as well, as a separate application. Isn't that important?

Reply:
No, it's deployed as a static build so this doesn't matter.